velox/docs/monthly-updates/february-2023.rst

```diff
@@ -0,0 +1,77 @@
+********************
+February 2023 Update
+********************
+
+
+Core Library
+============
+
+ * Add several memory related fixes: task reference in partition output iobuf to prevent memory leaking,
+   memory corruption caused by ntile.
+ * Add concurrent memory allocation benchmark.
+ * Add various enhancements such as replace ArrayBuilder class with ArrayWriter, support for decimal types in DecodedVector,
+  make ConstantTypedExpr safer to use, Require type in FlatVector constructor, support to serialize and deserialize long decimals,
+  BloomFilter to support serialization and memory tracking
+ * Add fixes for null handling of decimal values in QueryAssertions, Decimal Sum Aggregate,
+ * Remove usage of variant::inferType from variantArrayToVector function, variant::inferType from core::ConstantExpr, BaseVector::isConstant(rows) method
+ * Add fixes for expression compilation and common sub-expression evaluation
+ * Add ArrowStream operator and support date type in Arrow bridge
+ * Add support for k PRECEDING/FOLLOWING window frames in ROWS mode
+
+
+Presto Functions
+================
+ * Add :func:`array_normalize` function.
+ * Add support for decimal inputs to least and greatest functions.
+
+
+Spark Functions
+===============
+
+ * Add sha1 SparkSQL function.
+ * Add documentation for Spark functions.
+
+
+Hive Connector
+==============
+
+ * Add decimal type support to RowContainer.
+ * Fix inconsistent dictionary vector type from `StringColumnReader` in the native Parquet reader.
+ * Add support for BOOLEAN in native Parquet reader and DATE type in HiveTypeParser.
+ * Add split and stripe prefetch in reader.
+
+
+Substrait
+=========
+ * Update substrait to 0.23.0.
+ * Add support for emit and for parsing Date fields in Substrait plans.
+
+
+Performance and Correctness
+===========================
+
+ * Add Cast in Fuzzer along with various bug fixes related to Cast.
+ * Add Try support in the Fuzzer.
+ * Add FuzzerConnector , which generates fuzzed data for query plans.
+ * Add enhancements to favorably select certain functions , generate repro files for window fuzzers, nested expression
+   re-use.
+ * Add support for decimal types to VectorFuzzer.
+ * Add support for GeneratorSpec.
+
+
+Build System
+============
+
+ * Add support to build PyVelox wheels for Linux and MacOS.
+ * Add benchmark result comparisons and reruns during Conbench CI runs.
```

Comment:
Can we ask Conbench folks to help with wording? I'm not sure I understand this item to suggest better phrasing. Can you explain what this is about?

Reply:
I think this is support for comparing various benchmarks and then supporting reruns (where we rerun those benchmarks a few times). I will reach out to conbench folks and get better wording.